requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.15.1
-scipy==1.0.1
-ortools==6.9.5824
-packaging==16.8
\ No newline at end of file
+numpy>=1.15.1
```

Comment:
Let's leave these as-is or bump to the latest version. `requirements` are for pinning the versions so that the build is reproducible.